src/main/java/pages/BookingConfirmationPage.java

```diff
@@ -0,0 +1,20 @@
+package pages;
+
+import utils.Helper;
+
+public class BookingConfirmationPage extends BasePage {
+
+	Helper helper = new Helper();
+	
+	String filePathBookingConfirmationPage = "./src/main/resources/objectRepo/BookingConfirmationPage.json",
+			successfulTextClassName = helper.parseJSONToString("successfulTextClassName", filePathBookingConfirmationPage);
```

Comment:
successfulTextClassName should be bookingSuccessfulTextClassName to make it more clear what we are doing at each step.